modules/update-other-images-40.adoc

```diff
@@ -61,19 +61,34 @@ $ oc -n stackrox patch deploy/sensor -p '{"spec":{"template":{"spec":{"container
 +
 [source,terminal,subs=attributes+]
 ----
-$ oc -n stackrox set image deploy/sensor sensor=registry.redhat.io/rh-acs/main:{rhacs-version}
+$ oc -n stackrox set image deploy/sensor sensor=registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version}
 ----
 . Update the Compliance image:
 +
 [source,terminal,subs=attributes+]
 ----
-$ oc -n stackrox set image ds/collector compliance=registry.redhat.io/rh-acs/main:{rhacs-version}
+$ oc -n stackrox set image ds/collector compliance=registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version}
 ----
 . Update the Collector image:
 +
 [source,terminal,subs=attributes+]
 ----
-$ oc -n stackrox set image ds/collector collector=registry.redhat.io/rh-acs/collector:{collector-version}
+$ oc -n stackrox set image ds/collector collector=registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8:{rhacs-version}
+----
++
+[NOTE]
+====
+If you are using the collector slim image, run the following command instead:
```

Comment:
I think it's clearer if you present multiple options like this in parallel instead of putting one in a note, like:

~~~
. <Do a thing> in one of the following ways:
** If <condition>, run the following command:
+
----
----
** If <other_condition>, run the following command:
+
----
----
~~~

I see that you added a couple of these, so if you take this suggestion, please implement it across the PR.